pyodide-build/pyodide_build/tests/test_common.py

```diff
@@ -249,29 +248,23 @@ def test_repack_zip_archive(
     assert input_path.stat().st_size == expected_size
 
 
-def test_set_build_environment(monkeypatch):
+def test_get_build_environment_vars_host_env(monkeypatch):
     import os
 
-    monkeypatch.delenv("PKG_CONFIG_PATH", raising=False)
-    monkeypatch.delenv("WASM_PKG_CONFIG_PATH", raising=False)
```

Comment:
I think we should keep a test that there is no error when both of these are undefined.

Reply:
Actually, I realized that we don't need to separate WASM_PKG_CONFIG_PATH and PKG_CONFIG_PATH anymore since `get_build_environment_vars` does not pollute host environment variables anymore, and we have a logic that the host environment variable takes precedence over variables in Makefile.envs.